pages/dkp/kommander/2.2/insights/insights-setup/index.md

```diff
@@ -36,7 +36,7 @@ To receive insight items on the Insights Dashboard, enable the DKP Insights Engi
     EOF
     ```
 
-    This grants all Attached clusters the ability to enable the Insights Engine, and DKP Insights icon appears under Applications.
+    This grants all Attached clusters the ability to enable the Insights Engine, and displays DKP Insights under Applications.
```

Comment:
I'm just not sure what the correct subject verb agreement is here - so making it to display

```suggestion
    This grants all Attached clusters the ability to enable the Insights Engine, and to display DKP Insights under Applications.
```